index.html

```diff
@@ -290,22 +290,11 @@ <h6>JWT Encoding</h6>
           </p>
 
           <p class="note">
-            Implementers are warned that JWTs are not capable of encoding
-            multiple
-            <a>subjects</a> and are thus not capable of encoding a
-            <a>verifiable credential</a> with more than one <a>subject</a>. JWTs
-            might support multiple subjects in the future and implementers are
-            advised to refer to the
-            <a href="https://www.iana.org/assignments/jwt/">
-              JSON Web Token Claims Registry</a
-            >
-            for multi-subject JWT claim names or the
-            <a
-              href="https://datatracker.ietf.org/doc/draft-yusef-oauth-nested-jwt/"
-            >
-              Nested JSON Web Token</a
-            >
-            specification.
+	    At the time of this writing, JWTs do not have a representation for multiple subjects
+	    and are thus not capable of encoding a verifiable credential with more than one subject.
+	    Should the working group decide to do so,
+	    we could create a new JWT claim for this purpose and register it in the 
+            <a href="https://www.iana.org/assignments/jwt/">JSON Web Token Claims Registry</a>.
```

Comment:
@selfissued — This document is not aimed at the working group, so it seems odd to change this *note* to be so aimed. It is even more odd to remove all the `<a>...</a>` reference wrappers. 

@OR13, @Sakurann — Perhaps (one of) you might suggest appropriate revisions to the PR, since you approved the wording (which didn't include the ReSpec markup) in the connected issue (#24)?

Reply:
I'm not sure what you are requesting, I think the pull request is an improvement over the current text.

Reply:
"Should the working group decide to do so, we could create..." doesn't make sense in text aimed at people outside the WG.
```suggestion
	    The Working Group may decide to create a new JWT claim for
	    this purpose, and, if so, may register it in the
            <a href="https://www.iana.org/assignments/jwt/">JSON Web Token Claims Registry</a>.
```